packages/antd/test/Array.test.js

```diff
@@ -76,3 +76,60 @@ describe("array fields", () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe("errors", () => {
```

Comment:
If you delete these three lines, then they will all be in a single describe block, which is what I was hoping for